homeassistant/helpers/config_validation.py

```diff
@@ -1106,8 +1106,10 @@ def STATE_CONDITION_SCHEMA(value: Any) -> dict:  # pylint: disable=invalid-name
     )
 )
 
+TRIGGER_BASE_SCHEMA = {vol.Required(CONF_PLATFORM): str}
+
 TRIGGER_SCHEMA = vol.All(
-    ensure_list, [vol.Schema({vol.Required(CONF_PLATFORM): str}, extra=vol.ALLOW_EXTRA)]
+    ensure_list, [vol.Schema({**TRIGGER_BASE_SCHEMA}, extra=vol.ALLOW_EXTRA)]
```

Comment:
Normally we use `Schema.extend`. Is there a reason we don't use that here?

Reply:
You're right, should I change to use that throughout?

Reply:
I think so. :+1: